drivers/i2c/i2c_ite_enhance.c

```diff
@@ -0,0 +1,771 @@
+/*
+ * Copyright (c) 2022 ITE Corporation. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT ite_enhance_i2c
+
+#include <drivers/gpio.h>
+#include <drivers/i2c.h>
+#include <drivers/pinmux.h>
+#include <errno.h>
+#include <logging/log.h>
+LOG_MODULE_REGISTER(i2c_ite_enhance, CONFIG_I2C_LOG_LEVEL);
```

Comment:
for consistency, can this be put after include list?

Reply:
Done.